2024/day31/minikube_setup.md

```
@@ -0,0 +1,82 @@
+    sudo apt-get update
+    ```
+    ```
+    sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+    ```
+ - add user ubuntu to docker group
+    ```
+    sudo usermod -aG docker $USER && newgrp docker 
+---
+
+ ### Install Minikube & kubectl 
+ - Download and install minikube
```

Markdownlint

Expected: 0; Actual: 1
Unordered list indentation

(MD007, ul-indent)